toolkit/tools/pkg/imagecustomizerlib/extractpartitions.go

```diff
@@ -240,6 +241,7 @@ func constructOutputPartitionMetadata(diskPartition diskutils.PartitionInfo, par
 	partitionMetadata.Uuid = diskPartition.Uuid
 	partitionMetadata.PartUuid = diskPartition.PartUuid
 	partitionMetadata.Mountpoint = diskPartition.Mountpoint
+	partitionMetadata.OriginalSize = diskPartition.SizeInBytes
```

Comment:
This line is wrong. The partition has already been shrunk earlier during OS customization, but `originalSize` should hold the value from before partition shrinking.